datalake/src/main/java/com/sequenceiq/datalake/events/DatalakeUrlParser.java

```diff
@@ -0,0 +1,48 @@
+package com.sequenceiq.datalake.events;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.structuredevent.rest.urlparser.CDPRestUrlParser;
+
+// todo: https://jira.cloudera.com/browse/CB-13786 consider renaming to BackupRestoreParser
+@Component
+public class DatalakeUrlParser extends CDPRestUrlParser {
+    // todo: https://jira.cloudera.com/browse/CB-13786 implement me
+    @Override
+    protected Pattern getPattern() {
+        return null;
```

Comment:
Id `auditEnabled=true` it will throw an NPE. Eighter uncomment the filter registration or you should write a fake regexp here.

Reply:
I've commented out the usage in the filter registration, and added a todo referencing CB-13786.